R/navbar.R

```diff
@@ -200,6 +200,9 @@ bs4_navbar_links_tags <- function(links, depth = 0L) {
   # function for links
   tackle_link <- function(x, index, is_submenu, depth) {
 
+    target = NULL
+    if (!is.na(x$target) && !is.null(x$target)) target =  x$target
```

Comment:
Why might `target` be either `NULL` or `NA`?

Reply:
Sorry, I realize now that it's unnecessary to check that. The following commit removes these lines: https://github.com/JSchoenbachler/pkgdown/commit/a0b8b5677470bc63c76ddaeaa49259eaab98712b